doc/LanguageClient.txt

```diff
@@ -32,18 +32,18 @@ At this point, call any provided function as you like. See
 |LanguageClientFunctions| for a complete list of functions. Usually one would
 like to map these functions to shortcuts, for example: >
```

Comment:
Oh yeah that's a good point!

Maybe we could do something like this?
e.g.
``` vim
At this point, call any provided function as you like. See
|LanguageClientFunctions| for a complete list of functions. Usually one would
like to map these functions to shortcuts, for example: >

    nnoremap <silent> K :call LanguageClient#textDocument_hover()<CR>
    nnoremap <silent> gd :call LanguageClient#textDocument_definition()<CR>
    nnoremap <silent> <F2> :call LanguageClient#textDocument_rename()<CR>

You can also use slightly cleaner Plug mappings instead of the full function name: >

    nmap <silent> K <Plug>(lcn-hover)
    nmap <silent> gd <Plug>(lcn-definition)
    nmap <silent> <F2> <Plug>(lcn-rename)
```
Please suggest better wordings of the "or use Plug mappings" phrase if you have any! Thanks!

Reply:
That looks good to me :+1: